client/thin-replica-client/include/client/thin-replica-client/grpc_connection.hpp

```diff
@@ -77,17 +78,29 @@ class GrpcConnection {
   // Possible results of RPC operations a GrpcConnection may report.
   enum class Result { kUnknown, kSuccess, kFailure, kTimeout, kOutOfRange, kNotFound, kEndOfStream };
 
+  static const size_t maxWorkerThreadPoolSize = 8;
```

Comment:
Why did we choose size 8 for max pool size?

Reply:
The default size is 1 thread. The max is to guard against any misuse. No specific reason for 8 whoever tunes it in future can bump it as required.